BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -137,16 +137,26 @@
     <string name="format_options">Format Options</string>
     <string name="format_arrow">Format Arrow</string>
 
-
     <!-- Export Dialog Alert Window Descriptions -->
     <string name="storage_permission_needed">Zugriff erforderlich</string>
     <string name="storage_permission_alert_msg">
         Zugriff auf den Gerätespeicher erforderlich,
         um Dateien aus dieser App zu exportieren.
+        Erlaube den Zugriff und versuche nochmal die
+        Dateien zu exportieren.
     </string>
-    <string name="storage_permission_ok_btn">OK</string>
+    <string name="ok">OK</string>
     <string name="storage_permission_cancel_btn">Abbrechen</string>
     <string name="storage_permission_denied">Zugriff verweigert</string>
+    <string name="file_write_failed">File write failed:</string>
```

Comment:
This should be German, how about: "Speichern fehlgeschlagen:"

Reply:
Since this is a part of the exception and is not shown to the user, we do not need to translate it.

Reply:
Our app is in German, so all texts should be... If the text is not even needed, I suggest deleting it.

Reply:
When I delete it, then I have a checkstyle error.
![image](https://user-images.githubusercontent.com/37459401/109397417-bb26a900-7936-11eb-9b47-40cbecbbf41f.png)

Reply:
Well, if it is need, please translate to German :)

Reply:
This is used only by the programmer and is not related to the UI of the app. Since we write in English I would leave it in English.